server/src/models/UserPersonalization.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryBuilder } from 'objection';
+
+import { Domain } from './Domain';
+import { User } from './User';
+import { withDates } from './utils';
+
+export class UserPersonalization extends withDates(Domain) {
+  id!: string;
+  user_id!: string;
+
+  type!: string | null;
+  value!: string | null;
+
+  user!: User;
+
+  static tableName = 'omscentral_user_personalization';
+
+  static relationMappings = {
+    user: {
```

Comment:
user should be userPersonalization to match the table defined in from field.
This is necessary to resolve the attached error:
[BelongsToOneRelation stack trace Dec 26.txt](https://github.com/ghormadesoham/omscentral/files/7780233/BelongsToOneRelation.stack.trace.Dec.26.txt)

Reply:
Done!